api/tests/unit/domain/services/user-reconciliation-service_test.js

```diff
@@ -602,4 +573,186 @@ describe('Unit | Service | user-reconciliation-service', () => {
       expect(result).to.not.equal(originaldUsername);
     });
   });
+
+  describe('#checkIfStudentHasAnAlreadyReconciledAccount', () => {
+
+    let userRepositoryStub;
+    let obfuscationServiceStub;
+    let studentRepositoryStub;
+
+    beforeEach(() => {
+      userRepositoryStub = { getUserAuthenticationMethods: sinon.stub() };
+      obfuscationServiceStub = { getUserAuthenticationMethodWithObfuscation: sinon.stub() };
+      studentRepositoryStub = { getReconciledStudentByNationalStudentId: sinon.stub() };
+    });
+
+    context('When student is already reconciled in the same organization', () => {
+
+      context('When the reconciled account as an email', () => {
```

Comment:
```suggestion
      context('When the reconciled account has an email', () => {
```
La correction est à appliquer dans chaque nom de test du fichier

Reply:
Done.